webpack_loader/loader.py

```diff
@@ -53,9 +65,15 @@ def map_chunk_files_to_url(self, chunks):
         assets = self.get_assets()
         files = assets['assets']
 
+        add_integrity = self.config['INTEGRITY']
```

Comment:
To keep backwards compatibility, I think this should be `self.config.get('INTEGRITY', False)`